k8s/translation.go

```diff
@@ -179,6 +179,66 @@ func marshalJSON(obj resource.Object, extraLabels map[string]string, cfg ClientC
 	return json.Marshal(co)
 }
 
+var metaV1Fields = getV1ObjectMetaFields()
```

Comment:
Done dynamically once with reflection, instead of hard-coding the v1meta.ObjectMeta fields (can't use an empty JSON-marshaled object for keys, because they nearly all have `,omitempty` on their JSON tags).